packages/backend/src/config/app.ts

```diff
@@ -123,17 +126,40 @@ function parseRedisTlsConfig(
 
   // self-signed certs.
   if (caFile !== '') {
-    options.ca = readFileSync(caFile)
+    options.ca = fs.readFileSync(caFile)
     options.rejectUnauthorized = false
   }
 
   if (certFile !== '') {
-    options.cert = readFileSync(certFile)
+    options.cert = fs.readFileSync(certFile)
   }
 
   if (keyFile !== '') {
-    options.key = readFileSync(keyFile)
+    options.key = fs.readFileSync(keyFile)
   }
 
   return Object.keys(options).length > 0 ? options : undefined
 }
+
+function parseOrProvisionKey(keyPath: string): crypto.KeyObject {
```

Comment:
Is it worth adding a `config.test.ts` for this?

Reply:
Probably yes 👍